_monitoring-plugins/ad/api.md

```diff
@@ -2907,6 +2918,8 @@ Introduced 1.1
 
 Deletes the results of a detector based on a query.
 
+This command doesn't delete any custom result indices. You need to manually delete any custom result indices that you don’t need anymore.
```

Comment:
This delete results API is to delete anomaly result documents in default result index. It doesn't support deleting any **anomaly results** in custom result indices. You need to manually delete any **anomaly results** in custom result indices that you don’t need anymore.

Note: this API is not to delete the whole index, just delete anomaly results in the index